### DISCUSSION TITLE
Is it possible to use latex package in html without change any setting?

### DISCUSSION THREAD

**Discussion by Lightup1** [quarto-dev/quarto-cli#1620] [Q&A] [answered]
For rendering pdf, it is possible to use latex package.
For example, one can use pacage `physics` and write equation like \qty(x^2) and get $(x^2)$
But for html, it just showing equation like this $\qty(x^2)$.
Is there a way to let html use latex package without modifying every equation?

**Comment by mcanouil** [comment, marked as answer]
You have to use MathJax, KaTeX, etc. as described in the documentation: <https://quarto.org/docs/output-formats/html-basics.html#latex-equations>

There is an example in the [gallery](https://quarto.org/docs/gallery/#articles-reports), namely ["HTML for web publishing"](https://quarto-dev.github.io/quarto-gallery/articles/html/html.html) using KaTeX (see the code at <https://github.com/quarto-dev/quarto-gallery/blob/main/articles/html/html.qmd>).

The tricky part is to find which HTML library provides `\qty`.

```
---
title: "Quarto HTML Basics"
format:
  html:
    html-math-method: katex
---

$$
\qty(x^2)
$$

```

**Reply by Lightup1** [reply]
Thanks.
But how to add MahtJax extensions in Quarto?
For example, add `Physics` http://docs.mathjax.org/en/latest/input/tex/extensions/physics.html?highlight=physics?

**Reply by Lightup1** [reply]
Thanks. 
Seems katex just support \qty.

**Reply by Lightup1** [reply]
But generally, is it possible to let Quarto support defining MathJax extensions?

**Reply by Lightup1** [reply]
I tried that
```
---
title: "Quarto HTML Basics"
format:
  html:
    html-math-method: katex
---

$$
\qty(x^2)
$$
```
I just got this.
![image](https://user-images.githubusercontent.com/32999017/182138296-9da8bb23-2334-4dd7-b251-3d6a81ee9f52.png)

**Reply by mcanouil** [reply]
As described in the link you provided (*i.e.*, <http://docs.mathjax.org/en/latest/input/tex/extensions/physics.html?highlight=physics>) you have to change MathJax configuration locally or using a file.
Something like below should work (not tested).

```
---
title: "Quarto Math"
format:
  html:
    html-math-method: mathjax
    include-in-header:
      - text: |
          <script>
          window.MathJax = {
            loader: {
              load: ['[tex]/physics']
            },
            tex: {
              packages: {'[+]': ['physics']}
            }
          };
          </script>
---

$$
\qty(x^2)
$$
```

**Reply by Lightup1** [reply]
Thanks this one works!
